### PR TITLE
fix(core): Use primordials for methods

### PR DESCRIPTION
**Pull request by petamoriken**
I would like to get this change into Deno before merging https://github.com/denoland/deno_lint/pull/1152

### CODE REVIEW

File: ext/web/06_streams.js [hunk 1]
```diff
@@ -1252,7 +1253,16 @@ function readableByteStreamControllerEnqueueClonedChunkToQueue(
 ) {
   let cloneResult;
   try {
-    cloneResult = buffer.slice(byteOffset, byteOffset + byteLength);
+    if (ObjectPrototypeIsPrototypeOf(buffer, ArrayBufferPrototype)) {
```
Comment (petamoriken) [inline ext/web/06_streams.js:1256]: My mistake

File: ext/console/02_console.js [hunk 2]
```diff
@@ -7,117 +7,121 @@ const internals = globalThis.__bootstrap.internals;
...
-  ReflectGet,
-  ReflectGetOwnPropertyDescriptor,
-  ReflectGetPrototypeOf,
-  ReflectHas,
+  TypeError,
   TypedArrayPrototypeGetLength,
   TypedArrayPrototypeGetSymbolToStringTag,
   WeakMapPrototype,
   WeakSetPrototype,
+  isNaN,
```
Comment (petamoriken) [inline ext/console/02_console.js:124]: Sorted mechanically.

File: ext/fs/30_fs.js [hunk 3]
```diff
@@ -369,7 +372,7 @@ async function link(oldpath, newpath) {
 
 function toUnixTimeFromEpoch(value) {
   if (ObjectPrototypeIsPrototypeOf(DatePrototype, value)) {
-    const time = value.valueOf();
+    const time = DatePrototypeGetTime(value);
```
Comment (petamoriken) [inline ext/fs/30_fs.js:375]: `Date#valueOf` and `Date#getTime` return the same value. Either is acceptable, but I chose the latter for clarity.

**Comment by bartlomieju** [comment]
@petamoriken sorry I didn't merge it before the release. Could you please rebase this PR again?

File: ext/console/01_console.js [hunk 4]
```diff
@@ -2154,7 +2177,7 @@ function formatMapIterInner(
     // retrieved ones exist, we can not reliably return the same output) if the
     // output is not sorted anyway.
     if (!ctx.sorted) {
-      output = output.sort();
+      ArrayPrototypeSort(output);
```
Comment (petamoriken) [inline ext/console/01_console.js:2180]: There is no need to reassign.

**Comment by petamoriken** [comment]
@bartlomieju I've resolved conflicts!

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks

**Pull request merged**